pkg/network/setup/netpod/discover.go

```diff
@@ -96,8 +96,14 @@ func (n NetPod) discover(currentStatus *nmstate.Status) error {
 				return err
 			}
 
-		// Skip the discovery for all other known network interface bindings.
 		case vmiSpecIface.Binding != nil:
+			if podIfaceExists {
```

Comment:
The logic here is different from the other handled bindings.
I think it requires an explanation (i.e. explain why it ignores validation of link existence).

Reply:
done.